index.js

```diff
@@ -244,13 +250,19 @@ function localizeAnimationShorthandDeclValues(decl, context) {
 
   const didParseAnimationName = false;
   let parsedAnimationKeywords = {};
+  let stepsFunctionNode = null;
   const valueNodes = valueParser(decl.value).walk((node) => {
     /* If div-token appeared (represents as comma ','), a possibility of an animation-keywords should be reflesh. */
     if (node.type === 'div') {
       parsedAnimationKeywords = {};
     }
+    if (node.type === 'function' && node.value === 'steps') {
```

Comment:
`steps` can be uppercased like `STEPS()`

Reply:
Also please add test for `STEPS`

Reply:
good catch)